i18n/locales/en.json

```diff
@@ -80,6 +80,12 @@
             "modules": "Modules",
             "publishingNodeApiModules": "How to publish a Node-API package"
           }
+        },
+        "backpressuring": {
```

Comment:
The category should be "streams", not "Backpressuring". Can you make these changes? (Since it will require changes in many files?)

Also, the link should probably be /learn/streams/backpressuring-in-streams

Reply:
Actually, this guide should be in the "Modules" category 🤔